.github/workflows/backup-db.yml

```diff
@@ -98,54 +98,13 @@ jobs:
         db-server-name: ${{ inputs.db-server }}
         slack-webhook: ${{ secrets.SLACK_WEBHOOK }}
 
-    - name: Disk cleanup
-      if: github.event_name == 'schedule'
-      shell: bash
-      run: |
-        sudo rm -rf /usr/local/lib/android || true
-        sudo rm -rf /usr/share/dotnet || true
-        sudo rm -rf /opt/ghc || true
-        sudo rm -rf /usr/local/.ghcup || true
-        sudo rm -rf /opt/hostedtoolcache/CodeQL || true
-        sudo rm -rf /usr/local/share/boost || true
-        sudo docker image prune --all --force || true
-        sudo apt-get remove -y '^aspnetcore-.*' || true
-        sudo apt-get remove -y '^dotnet-.*' --fix-missing || true
-        sudo apt-get remove -y '^llvm-.*' --fix-missing || true
-        sudo apt-get remove -y 'php.*' --fix-missing || true
-        sudo apt-get remove -y '^mongodb-.*' --fix-missing || true
-        sudo apt-get remove -y '^mysql-.*' --fix-missing || true
-        sudo apt-get remove -y google-chrome-stable firefox powershell mono-devel libgl1-mesa-dri --fix-missing || true
-        sudo apt-get remove -y google-cloud-sdk --fix-missing || true
-        sudo apt-get remove -y google-cloud-cli --fix-missing || true
-        sudo rm -rf "$AGENT_TOOLSDIRECTORY"/PyPy || true
-        sudo rm -rf "$AGENT_TOOLSDIRECTORY"/Python || true
-        sudo rm -rf "$AGENT_TOOLSDIRECTORY"/go || true
-        sudo rm -rf "$AGENT_TOOLSDIRECTORY"/node || true
-        sudo apt-get autoremove -y || true
-        sudo apt-get clean
-
-    - name: Sanitise dump
-      if: github.event_name == 'schedule'
-      run: |
-        createdb ${DATABASE_NAME} && gzip -d --to-stdout ${{ env.BACKUP_FILE }}.sql.gz | psql -d ${DATABASE_NAME}
-        rm ${{ env.BACKUP_FILE }}.sql.gz
-        psql -d ${DATABASE_NAME} -f db/scripts/sanitise.sql
-        pg_dump --encoding utf8 --compress=1 --clean --no-owner --if-exists -d ${DATABASE_NAME} -f att_backup_sanitised.sql.gz
-      env:
-        DATABASE_NAME: apply_manage_itt
-        PGUSER: postgres
-        PGPASSWORD: postgres
-        PGHOST: localhost
-        PGPORT: 5432
-
-    - name: Upload sanitized backup to Azure storage
+    - name: Upload backup to Azure storage
```

Comment:
it's already uploaded in the previous step by github-actions/backup-postgres@master